### PR TITLE
gives proper light armor slowdown to goon armor

### PR DESCRIPTION
**Pull request by InsaneRed**
# About the pull request
Gives light_slowdown to the goon armor that spawns on survivors, since it has BETTER THEN LIGHTARMOR without any slowdown

# Explain why it's good for the game

People are powergaming with survivor armor and i dont like it, also the goon armor is described as being shady and bad. Doesnt make sense it has nearly medium stats with no slowdown at all.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: Goon armor properly has slowdown now
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/marine_uniform.dm [hunk 1]
```diff
@@ -492,6 +492,7 @@
 	contained_sprite = TRUE
 	item_state_slots = null
 	flags_jumpsuit = UNIFORM_SLEEVE_ROLLABLE
+	slowdown = SLOWDOWN_ARMOR_LIGHT
```
Comment (Diegoflores31) [inline code/modules/clothing/under/marine_uniform.dm:495]: This is the uniform brother wtf

Comment (InsaneRed) [reply code/modules/clothing/under/marine_uniform.dm]: whoops

File: code/modules/clothing/suits/marine_armor/ert.dm [hunk 2]
```diff
@@ -79,6 +79,7 @@
 	item_state = "armor"
 	item_state_slots = null
 	contained_sprite = TRUE
+	slowdown = SLOWDOWN_ARMOR_LIGHT
```
Comment (Diegoflores31) [inline code/modules/clothing/suits/marine_armor/ert.dm:82]: This armor already has slowdown by inheritance from pmc/light armor . There is no need to write it again

Comment (Diegoflores31) [inline code/modules/clothing/suits/marine_armor/ert.dm:82]: I would like to add this armor only covers the CHEST . And has pretty much the same or less stats than a jacket .  Also you are adding the slowdown variable wrong

**Comment by InsaneRed** [comment]
i'll be adding a few more changes soon

**Comment by InsaneRed** [comment]
after some testing, it doesnt need anymore tweaking as the things i have been told were incorrect and it already has neuro stun resistance (from sentinel)

File: code/modules/clothing/under/marine_uniform.dm [hunk 3]
```diff
@@ -888,7 +888,6 @@
 	item_icons = list(
 		WEAR_BODY = 'icons/mob/humans/onmob/uniform_1.dmi',
 	)
-	
 /obj/item/clothing/under/marine/reporter/black
```
Comment (Drulikar) [inline code/modules/clothing/under/marine_uniform.dm:891]:
```suggestion

/obj/item/clothing/under/marine/reporter/black
```

**Review by Drulikar — APPROVED** [review]

**Comment by Skatuk** [comment]
While we’re on the goon armour point give it protection against sentinel neuro spit 👍

**Pull request merged**